src/bert.erl

```diff
@@ -35,7 +45,7 @@ encode_term(List) when is_list(List) ->
     % TODO: Handle improper lists
     lists:map(fun encode_term/1, List);
 
-encode_term(Tuple) when is_tuple(Tuple) ->
+encode_term(Tuple) when is_tuple(Tuple), Tuple =/={} ->
```

Comment:
Why is this extra guard is necessary?

Reply:
Oh, OK. `encode({})` throws if that guard is not present. Could you add a test for that?

Reply:
> Oh, OK. `encode({})` throws if that guard is not present. Could you add a test for that?

I already did it. There is in property based test. By using property based tests I quickly found that restriction. It is possible to add EUnit test too.